.gitignore

```diff
@@ -49,3 +49,6 @@ generated/
 # vscode configs
 .vscode/settings.json
 .vscode/c_cpp_properties.json
+txt.txt
```

Comment:
Ignore these files with a wildcard, like "*.txt", instead of writing each filename individually